tests/unit/plugins/lookup/onepassword/files/v1_out_01.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Please add a `.license` file for every JSON file containing something like
```
GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
SPDX-License-Identifier: GPL-3.0-or-later
SPDX-FileCopyrightText: 2022, Ansible Project
```
That should fix the failing sanity tests.

Reply:
Awesome. Will do!